src/client/api/SConscript

```diff
@@ -3,7 +3,7 @@ import daos_build
 
 def scons():
     """Execute build"""
-    Import('env', 'DAOS_VERSION', 'prereqs')
+    Import('env', 'DAOS_VERSION', 'prereqs', 'API_VERSION')
```

Comment:
Do we need DAOS_VERSION here?

Reply:
I would suppose not.  Fixed in next commit.